src/engine/engine.go

```diff
@@ -133,22 +133,26 @@ func (e *Engine) renderBlock(block *Block) {
 			if padText, OK := e.shouldFill(block, length); OK {
 				e.write(padText)
 			}
-			e.writeANSI(e.Ansi.CarriageForward())
-			e.writeANSI(e.Ansi.GetCursorForRightWrite(length, block.HorizontalOffset))
+			ansi := e.Ansi
+			if e.Env.Shell() == shell.BASH {
+				ansi = &color.Ansi{}
+				ansi.InitPlain()
+				e.writeANSI(`\[`)
+			}
+			e.writeANSI(ansi.CarriageForward())
+			e.writeANSI(ansi.GetCursorForRightWrite(length, block.HorizontalOffset))
 			e.currentLineLength = 0
 			e.write(text)
+			if e.Env.Shell() == shell.BASH {
+				e.writeANSI(`\[`)
```

Comment:
```suggestion
				e.writeANSI(`\]`)
```

I think this is a typo.